### ISSUE TITLE
UnboundLocalError: local variable 'action_server' referenced before assignment

### ISSUE THREAD

**Issue by hawesie** [strands-project/strands_navigation#218]
It might be my error, but here it is anyway. Action being triggered by nav_client.py. Robot is positioned on WayPoint1. If I move the robot off WayPoint1 and trigger it, it works.

```
[INFO] [WallTime: 1428694788.474106] Started SMACH action server wrapper, adversiting as 'monitored_navigation'
[INFO] [WallTime: 1428694788.579792]  ...done
[INFO] [WallTime: 1428694788.580061] Subscribing to Localisation Topics
[INFO] [WallTime: 1428694788.586255]  ...done
[INFO] [WallTime: 1428694788.586561] Creating Reconfigure Client
[INFO] [WallTime: 1428694788.599741] new node reached WayPoint1
[INFO] [WallTime: 1428694788.604115] All Done ...
[INFO] [WallTime: 1428694788.604134]  ...done
[INFO] [WallTime: 1428694788.604428] Subscribing to Localisation Topics
[INFO] [WallTime: 1428694788.610218]  ...done
[INFO] [WallTime: 1428694788.610570] All Done ...
[INFO] [WallTime: 1428694788.631983] new node reached WayPoint1
[INFO] [WallTime: 1428694807.166525] Navigating From WayPoint1 to WayPoint2
[INFO] [WallTime: 1428694807.166766] Waiting for Topological map ...
source: ['WayPoint1']
edge_id: ['WayPoint1_WayPoint2']
[INFO] [WallTime: 1428694807.167072] 1 Nodes on route
[INFO] [WallTime: 1428694807.167279] first action doorPassing
[ERROR] [WallTime: 1428694807.167739] Exception in your execute callback: local variable 'action_server' referenced before assignment
Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/site-packages/actionlib/simple_action_server.py", line 299, in executeLoop
    self.execute_callback(goal)
  File "/Users/nah/code/indigo_ws/src/strands_navigation/topological_navigation/scripts/navigation.py", line 156, in executeCallback
    self.navigate(goal.target)
  File "/Users/nah/code/indigo_ws/src/strands_navigation/topological_navigation/scripts/navigation.py", line 225, in navigate
    result, inc = self.followRoute(route, target)
  File "/Users/nah/code/indigo_ws/src/strands_navigation/topological_navigation/scripts/navigation.py", line 329, in followRoute
    if action_server is None:
UnboundLocalError: local variable 'action_server' referenced before assignment
```

**Comment by hawesie** [comment]
This is pretty repeatable.

**Comment by Jailander** [comment]
Ok, I'll take a look tomorrow

**Comment by Jailander** [comment]
hopefully fixed by #221

**Comment by Jailander** [comment]
I don't think it has been seen again will close reopen if needed
